src/Parser.js

```diff
@@ -46,8 +46,9 @@ var ParseError = require("./ParseError");
  * Main Parser class
  */
 function Parser(input, settings) {
-    // Make a new lexer
-    this.lexer = new Lexer(input);
+    // Make a new lexer (mouth) and macro expander (gullet)
+    // for this parser (stomach, in the language of TeX)
+    this.gullet = new MacroExpander(input, settings.macros);
```

Comment:
The comment is a bit confusing b/c it's the MacroExpander that creators the lexer.